buildSrc/src/main/kotlin/com/datadog/gradle/plugin/jsonschema/JsonSchemaReader.kt

```diff
@@ -27,9 +27,6 @@ class JsonSchemaReader(
     fun readSchema(schemaFile: File): TypeDefinition {
         logger.info("Reading schema ${schemaFile.name}")
         val schema = loadSchema(schemaFile)
-        require(schema.type == JsonType.OBJECT) {
```

Comment:
why this is removed if no changes are done to the reader logic?

Reply:
is to support the special case I added in the tests. Is because the browser team needed this in the schemas and we are sharing that with them.

Reply:
ok, just wanted to make sure things work as expected in the `JsonSchemaReader` for the `schema.type != JsonType.OBJECT` case

Reply:
Yes, apparently they work, I still need to ask Xavier why he added this there in the first place. Maybe it was a constraint in the beginning.